bin/www

```diff
@@ -4,10 +4,28 @@
  * Module dependencies.
  */
 
-var app = require('../app');
+var ErrorCat = require('error-cat');
 var debug = require('debug')('detention:server');
+var hasKeypaths = require('101/has-keypaths');
 var http = require('http');
 
+var app = require('../app');
+var log = require('../logger')(__filename);
+
+var requiredEnvKeys = [
+  'HELLO_RUNNABLE_GITHUB_TOKEN',
+  'API_HOSTNAME'
+];
+if (!hasKeypaths(process.env, requiredEnvKeys)) {
+  log.error({
+    requiredEnvKeys: requiredEnvKeys,
+    env: process.env
+  }, 'Missing required ENV keys');
+  // send to rollbar
+  ErrorCat.report(new Error('Detention missing required ENV values'));
```

Comment:
this error will not get sent to rollbar, call it in the callback of this funtion
ErrorCat.prototype.report = function (err, req, cb) {